### PR TITLE
Update Team-Composition.md

### PR DESCRIPTION
**Pull request by kniz-raft**
### Summary of Changes
_Removed former team members (Romelo, Spencer), added new ones (Jorge, Alex P)_

### CODE REVIEW

File: docs/How-We-Work/Team-Composition.md [hunk 1]
```diff
@@ -58,3 +57,5 @@
 * Timothy Spencer, 18F, consulting engineer
 * Abbey Kos, 18F, research consultant
 * Randy Hart, 18F, acquisition consultant
+* Spencer Hilvitz, Raft, front end engineer
```
Comment (lfrohlich) [inline docs/How-We-Work/Team-Composition.md:60]: @kniz-raft  can you move Spencer and Romelo to the top of this list as most recent?

File: docs/How-We-Work/Team-Composition.md [hunk 2]
```diff
@@ -1,25 +1,24 @@
 ## Primary Product Team
 **OFA**
 * Lauren Frohlich, OFA, product owner
+* Alexandra Pennington, OFA, tech lead
```
Comment (ADPennington) [inline docs/How-We-Work/Team-Composition.md:4]: :)

**Review by ADPennington — COMMENTED** [review]
@kniz-raft, i think after @lfrohlich 's suggestions are implemented, good to go.

File: docs/How-We-Work/Team-Composition.md (hunk 1 above)
Comment (kniz-raft) [reply docs/How-We-Work/Team-Composition.md]: Done! I also added John Willis to current team.

**Review by ADPennington — APPROVED** [review]
👍🏾

**Review by lfrohlich — APPROVED** [review]
Thanks, @kniz-raft !

**Comment by lfrohlich** [comment]
@kniz-raft  this is ready to go to HHS main

**Pull request merged**